src/Backend/Modules/ContentBlocks/Command/CreateContentBlockHandler.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Backend\Modules\ContentBlocks\Command;
+
+use Backend\Core\Engine\Model;
+use Backend\Modules\ContentBlocks\Entity\ContentBlock;
+use Backend\Modules\ContentBlocks\Event\ContentBlockCreated;
```

Comment:
no :) no need for that

> On 22 Aug 2016, at 14:46, Jeroen Desloovere <notifications@github.com> wrote:
> 
> In src/Backend/Modules/ContentBlocks/Command/CreateContentBlockHandler.php <https://github.com/forkcms/forkcms/pull/1632#discussion_r75670226>:
> 
> > @@ -0,0 +1,67 @@
> > +<?php
> > +
> > +namespace Backend\Modules\ContentBlocks\Command;
> > +
> > +use Backend\Core\Engine\Model;
> > +use Backend\Modules\ContentBlocks\Entity\ContentBlock;
> > +use Backend\Modules\ContentBlocks\Event\ContentBlockCreated;
> Should Event class names be added the "Event" postfix?
> ContentBlockCreatedEvent
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/forkcms/forkcms/pull/1632/files/b33276f0e6e129878ec4ffc8bc1c6c29c166f2ff#r75670226>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ADd13kWnNKhnvUBNcdNh2t_xZFV9PzQsks5qiZosgaJpZM4JMwmz>.
>